local-cli/automated-vs/utils/utility.js

```diff
@@ -0,0 +1,11 @@
+'use-strict';
```

Comment:
utility.js is not a great module name. Maybe cliOptions? Idk...

Reply:
👍